.buildkite/pipeline.yml

```diff
@@ -62,7 +64,7 @@ steps:
  - label: test deb source packages via docker
```

Comment:
Some "via docker" things do not use docker anymore :wink:

Reply:
:yao: